.github/workflows/cicd-terraform-dsd-iam.yml

```diff
@@ -0,0 +1,108 @@
+name: "♻️ Terraform DSD"
```

Comment:
I think this might need a bit more information. How about Terraform AWS DSD Account?